src/ert/config/queue_config.py

```diff
@@ -90,7 +90,14 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
                     f"Valid choices are {sorted(VALID_QUEUE_OPTIONS[queue_system])}."
                 )
             if values:
-                queue_options[queue_system].append((option_name, values[0]))
+                if option_name == "LSF_SERVER" and values[0].startswith("$"):
+                    raise ConfigValidationError(
+                        f"Invalid server name for QUEUE_OPTION LSF LSF_SERVER: "
+                        f"{values[0]}. Check for empty enviroment variable. "
```

Comment:
enviroment -> environment

"The LSF_SERVER is usually specified in the site-config file. "  - maybe more like something like this? 
"The LSF_SERVER value is usually provided by default in a site-configuration file. If you have no special needs you can safely remove it from your configuration"